Airwallex/Payment/Sources/Utils/Strings.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Strings.swift
```

Comment:
Naming of extension file should be Type+Modifier.swift, for example `String+Filter`.swift here or simply `String+Extensions`.